test/nuts/create.nut.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { join } from 'path';
+import { expect } from '@salesforce/command/lib/test';
+import { TestSession } from '@salesforce/cli-plugins-testkit';
+import { execCmd } from '@salesforce/cli-plugins-testkit';
+import { fs } from '@salesforce/core';
+import { Dictionary } from '@salesforce/ts-types';
+
+const apexManifest =
+  '<?xml version="1.0" encoding="UTF-8"?>\n' +
+  '<Package xmlns="http://soap.sforce.com/2006/04/metadata">\n' +
+  '    <types>\n' +
+  '        <members>GeocodingService</members>\n' +
+  '        <members>GeocodingServiceTest</members>\n' +
+  '        <members>PagedResult</members>\n' +
+  '        <members>PropertyController</members>\n' +
+  '        <members>SampleDataController</members>\n' +
+  '        <members>TestPropertyController</members>\n' +
+  '        <members>TestSampleDataController</members>\n' +
+  '        <name>ApexClass</name>\n' +
+  '    </types>\n' +
+  '    <version>52.0</version>\n' +
+  '</Package>';
+
+describe('force:source:manifest:create', () => {
+  let session: TestSession;
+
+  before(async () => {
+    session = await TestSession.create({
+      project: {
+        gitClone: 'https://github.com/trailheadapps/dreamhouse-lwc.git',
+      },
+      setupCommands: [
+        'sfdx force:org:create -f config/project-scratch-def.json --setdefaultusername --wait 10 --durationdays 1',
+        'sfdx force:source:deploy -p force-app',
```

Comment:
Why does the test need to create an org and deploy as part of setup?

Reply:
I figured that way it would validate that the generated manifest was would work with a deploy. SDR should be verifying that `getPackageXml()` returns a valid package... but I thought it would be pretty easy to do here too